### PR TITLE
feat: add "Back to content" missing translation

### PR DESCRIPTION
**Pull request by ggallon**
Hi,

Add "**Back to content**" missing translation in ToolsPanel.

Not being multilingual 😬. I used a translator for languages other than French.

### CODE REVIEW

**Comment by ali-alhaidary** [comment]
Thank you @ggallon , however, you need only to update main.json and translation contributors can take over for proper translation.

**Comment by ggallon** [comment]
Hi @ali-alhaidary,
Ok thanks, do you want me to restore the translations so that the contributors can take over?

**Comment by ali-alhaidary** [comment]
> Hi @ali-alhaidary, Ok thanks, do you want me to restore the translations so that the contributors can take over?

I prefer so, as using google translation for RTL and Asian languages usually gives funny things :-)

**Comment by ggallon** [comment]
It's good.
Yes, even in French, it can be surprising sometimes😅

File: packages/tldraw/src/components/ToolsPanel/BackToContent.tsx [hunk 1]
```diff
@@ -39,7 +41,9 @@ export const BackToContent = React.memo(function BackToContent() {
 
   return (
     <BackToContentContainer id="TD-Tools-Back_to_content" style={{ ...style }}>
-      <RowButton onClick={app.zoomToContent}>Back to content</RowButton>
+      <RowButton onClick={app.zoomToContent}>
+        {intl.formatMessage({ id: 'zoom.to.content' })}
```
Comment (judicaelandria) [inline packages/tldraw/src/components/ToolsPanel/BackToContent.tsx:45]: If it's an html element it's better to use 
<FormattedMesssage id="zoom.to.content" />

**Review by judicaelandria — CHANGES_REQUESTED** [review]
Thanks for your contribution @ggallon

**Comment by steveruizok** [comment]
Thanks!

**Comment by ggallon** [comment]
@steveruizok @judicaelandria 
Thanks you

**Pull request merged**